redis/src/cluster_async/mod.rs

```diff
@@ -625,6 +625,23 @@ impl From<ArcStr> for OperationTarget {
     }
 }
 
+/// Represents a node to which a `MOVED` or `ASK` error redirects.
+pub(crate) struct RedirectNode {
```

Comment:
Do we want to make this `#[derive(Clone, Debug)]`?

Reply:
Added